appserver/security/appclient.security/src/main/java/com/sun/enterprise/security/appclient/AppClientSecurityInfoImpl.java

```diff
@@ -101,59 +98,45 @@ public void initializeSecurity(
             J2EESecurityManager mgr = new J2EESecurityManager();
             System.setSecurityManager(mgr);
         }
-        LOG.log(Level.DEBUG, "SEC9002: ACC: Security Manager is {0}", secMgr);
+        LOG.log(DEBUG, "SEC9002: ACC: Security Manager is {0}", secMgr);
 
-        //set the parser to ConfigXMLParser
+        // Set the parser to ConfigXMLParser
         System.setProperty("config.parser", DEFAULT_PARSER_CLASS);
         util.setAppClientMsgSecConfigs(msgSecConfigs);
+
         try {
-            /* setup jsr 196 factory
-             * define default factory if it is not already defined
-             */
+            // Setup Jakarta Authentication factory define default factory if it is not already defined
             String defaultFactory = java.security.Security.getProperty(DEFAULT_FACTORY_SECURITY_PROPERTY);
-            LOG.log(Level.DEBUG,
-                "AuthConfigFactory obtained from java.security.Security.getProperty(\"authconfigprovider.factory\"): {0}",
-                defaultFactory);
+            LOG.log(DEBUG, "AuthConfigFactory obtained from java.security.Security.getProperty(\"authconfigprovider.factory\"): {0}", defaultFactory);
             if (defaultFactory == null) {
-                java.security.Security.setProperty(DEFAULT_FACTORY_SECURITY_PROPERTY,
-                    GFAuthConfigFactory.class.getName());
+                java.security.Security.setProperty(DEFAULT_FACTORY_SECURITY_PROPERTY, AuthConfigFileFactory.class.getName());
             }
-
         } catch (Exception e) {
-            LOG.log(Level.WARNING, "SEC9001: ACC: Error in initializing JSR 196 Default Factory", e);
+            LOG.log(WARNING, "SEC9001: ACC: Error in initializing JSR 196 Default Factory", e);
         }
 
-        //TODO:V3 LoginContextDriver has a static variable dependency on AuditManager
-        //And since LoginContextDriver has too many static methods that use AuditManager
-        //we have to make this workaround here.
-        //Handles in LoginContextDriver
-        //LoginContextDriver.AUDIT_MANAGER = secServUtil.getAuditManager();
-
-        //secServUtil.initSecureSeed();
-
-        setSSLData(this.getTargetServers());
+        setSSLData(getTargetServers());
         if (username != null || password != null) {
             UsernamePasswordStore.set(username, password);
         }
 
-        //why am i setting both?.
```

Comment:
Don't comments without a value, please - even for commenting comments without a value ... maybe you could add FIXME ;-)

Reply:
Commenting on comments without a value. Sometimes there's a little bit of humour in the misery. I'm not sure if there is anything to be fixed though, maybe only the comment itself needs fixing.